recipes/pybedtools/0.6.9/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: pybedtools
+  version: '0.6.9'
+
+source:
+  url: https://github.com/daler/pybedtools/archive/v0.6.9.tar.gz
+  fn: v0.6.9.tar.gz
+  md5: a24e4bcd0c89beb9535295db964f6a4a
+
+build:
+  number: 0
+  skip: True # [py3k]
+
+requirements:
+  build:
+    - python
+    - six
+    - samtools
+    - setuptools
+    - pandas
+    - cython
+    - gcc  # [linux]
+    - llvm # [osx]
+    - zlib {{CONDA_ZLIB}}*
+
```

Comment:
The package does include a couple of shared libraries that are linked against zlib, so I went ahead and added the pinning for the build/run reqs.